cdap-distributions/src/etc/cdap/conf.dist/logback-container.xml

```diff
@@ -30,6 +30,12 @@
   <logger name="org.spark-project" level="WARN"/>
   <logger name="org.apache.hadoop" level="WARN"/>
   <logger name="org.apache.hive" level="WARN"/>
+  <logger name="org.apache.tephra.distributed.AbstractClientProvider" level="WARN"/>
+  <logger name="org.mortbay.log" level="WARN"/>
+  <logger name="SecurityLogger.org.apache.hadoop.security.authorize.ServiceAuthorizationManager" level="WARN"/>
```

Comment:
Yes, here is the snippet from the code: 
`  
/**
   * Configuration key for controlling service-level authorization for Hadoop.
   * 
   * @deprecated Use
   *             {@link CommonConfigurationKeys#HADOOP_SECURITY_AUTHORIZATION}
   *             instead.
   */
  @Deprecated
  public static final String SERVICE_AUTHORIZATION_CONFIG = 
    "hadoop.security.authorization";
  
  public static final Log AUDITLOG =
    LogFactory.getLog("SecurityLogger."+ServiceAuthorizationManager.class.getName());
`